fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckNetworkMiddleware.kt

```diff
@@ -87,11 +87,15 @@ class ReviewQualityCheckNetworkMiddleware(
                         return@launch
                     }
 
-                    val status = pollForAnalysisStatus()
+                    val statusProgress = pollForAnalysisStatus {
```

Comment:
We'll have to rebase here, this function has been extracted on it's own in an other PR.